cmd/commands/client/command_client.go

```diff
@@ -100,7 +100,7 @@ func (cmd *Command) Wait() error {
 func (cmd *Command) Kill() error {
 	err := cmd.connectionManager.Disconnect()
 	if err != nil {
-		return err
+		fmt.Printf("Disconnect error: %s\n", err)
```

Comment:
In case of disconnect when no connection exists, we return NoConnection error, we cannot make a return here, because it will not stop application itself. Just print message and continue

Reply:
According to our discussion. Special case with NoConnection is needed to handle here and otherwise pass error upstairs.

Reply:
Fixed.

Reply:
I agree stopp yelling in validation errors.
But still where are serious errors and they'are ignored

Reply:
Please check updated version